assets/js/icons/library/arrow-down-alt2.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * External dependencies
+ */
+import { SVG } from 'wordpress-components';
+
+const arrowDownAlt2 = ( { className, size, ...extraProps } ) => {
+	const iconClass = [ 'dashicon', 'dashicons-arrow-down-alt2', className ]
+		.filter( Boolean )
+		.join( ' ' );
```

Comment:
why not use classnames, a dependency we already use (that is quite lightweight).

Reply:
+1 I have same qu

Reply:
Because I copy/pasted from gb for this blindly :).  I'll add.